docs/ways-you-can-help-us.md

```diff
@@ -68,17 +68,19 @@ that works for you.
 
 ### Become a Dasharo Supporter
 
-Buy the Dasharo Supporters Entrance subscription if you own/use/buy a machine
-running Dasharo firmware:
+Buy the Dasharo Entry Subscription if you own/use/buy a machine running
+Dasharo firmware:
```

Comment:
Why are we limiting it to desktops only? Don't we want to use this model in the future on other platforms as well (not necessarily desktops)?

Reply:
But also there will be different entries in the shop with different pricing. This section does not mention any prices @pietrushnic so not sure what I am supposed to change here

Reply:
Please write in plain text that it is for desktop. You are linking to a 1/2/3 years subscription as it would be standard for every possible market segment, which is not true.

```
Buy the Dasharo Entry Subscription  for Desktop
```

Or below

```
[1-year subscription for Desktop]
```

Może jeszcze poniżej. If you are interested in other market segments than Desktop, please get in touch with us with the specific hardware model you have in mind. - albo coś w ten deseń

Reply:
How about that @pietrushnic ? 6709d8547fd4052f53e3cd44f0e731a09c97867d